data-raw/NCRO.R

```diff
@@ -73,7 +77,7 @@ NCRO <- mutate(NCROX,
                 # Since NCRO records only in PST, convert to local time to correspond with the other surveys
                 Datetime = with_tz(mdy_hm(`Collection Date`, tz = "Etc/GMT+8"), tzone = "America/Los_Angeles"),
                 Date = date(Datetime)) %>%
-  filter(`Sample Type` == "Normal Sample") %>%
+  filter(`Sample Type` == "Normal Sample", !is.na(sign)) %>%
```

Comment:
What is the effect of this? Does it just remove rows with NA values in a variable? And does it only remove values for that variable since it's in long format, or does it impact values in other variables as well?